### PR TITLE
Update starboard.go.tmpl

### PR DESCRIPTION
**Pull request by dvoraknt**
-added $maxAge user variable to allow for age limit to be set for stars added to messages
-added delete feature for messages on starboard if originating message falls below the star threshold

**Please describe the changes this PR makes and why it should be merged:**

**Status**

- [x] Code changes have been tested on an instance of YAGPDB, or there are no code changes
- [x] I have read and followed the [contribution guide](../CONTRIBUTING.md)

### CODE REVIEW

File: fun/starboard.go.tmpl [hunk 1]
```diff
@@ -31,7 +32,9 @@
 	{{ end }}
 {{ end }}
 
-{{ if and $count (or .ReactionMessage.Content .ReactionMessage.Attachments) (eq .Reaction.Emoji.Name $starEmoji) }}
+{{ if lt $count $starLimit }}
+	{{ deleteMessage $starboard $starboardMessage 0 }}
+{{ else if and $count (or .ReactionMessage.Content .ReactionMessage.Attachments) (eq .Reaction.Emoji.Name $starEmoji) (le (currentTime.Sub .Message.Timestamp.Parse) (toDuration $maxAge))}}
```
Comment (l-zeuch) [inline fun/starboard.go.tmpl:37]:
```suggestion
{{ else if and $count (or .ReactionMessage.Content .ReactionMessage.Attachments) (eq .Reaction.Emoji.Name $starEmoji) ((.Message.Timestamp.Parse.Add (toDuration $maxAge)).Before currentTime)}}
```
; )

**Review by l-zeuch — CHANGES_REQUESTED** [review]
LGTM, just this small change

File: fun/starboard.go.tmpl (hunk 1 above)
Comment (dvoraknt) [reply fun/starboard.go.tmpl:37]: :tomstare:
I guess I shouldn't be surprised

**Review by l-zeuch — APPROVED** [review]

**Comment by l-zeuch** [comment]
Thanks!

**Pull request merged**